### PR TITLE
Update allow-members-to-send-as-or-send-on-behalf-of-group.md

### PR DESCRIPTION
**Pull request by msbemba**
replaced members with users for the below reasons. verify and let me know if this is OK or should we just add a note about these two line items.

1) users can send emails as a group even without being a member of a group
2) not all members have access to send as and only internal users have that privilege.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4284

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Allow members to send as or send on behalf of a Group"
+title: "Allow Users to send as or send on behalf of a Group"
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
title: "Allow users to send as or send on behalf of a group"
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ ms.assetid: 0ad41414-0cc6-4b97-90fb-06bec7bcf590
 description: "Learn how to allow members to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."
 ---
 
-# Allow members to send as or send on behalf of a group
+# Allow Users to send as or send on behalf of a group
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
# Allow users to send as or send on behalf of a group
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ ms.assetid: 0ad41414-0cc6-4b97-90fb-06bec7bcf590
 description: "Learn how to allow members to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
description: "Learn how to allow users to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by MikePlumleyMSFT** [comment]
@msbemba "users" is too broad a term. Let's just note that guests aren't included near the top of the article.

**Review by Josegto1 — APPROVED** [review]

**Review by Josegto1 — APPROVED** [review]

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review.

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 4]
```diff
@@ -16,11 +16,14 @@ ms.collection:
 search.appverid:
 - MET150
 ms.assetid: 0ad41414-0cc6-4b97-90fb-06bec7bcf590
-description: "Learn how to allow members to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."
+description: "Learn how to allow users to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."
 ---
 
 # Allow members to send as or send on behalf of a group
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md [hunk 5]
```diff
@@ -16,11 +16,14 @@ ms.collection:
...
+> Guests who are members of the group cannot be granted to send as or send on behalf of a group
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md]:
```suggestion
> Guests who are members of a Microsoft 365 group cannot be granted **Send as** or **Send on behalf** permissions for the group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**